jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -283,10 +283,25 @@
     - shell: |
         set -x
         shopt -s extglob
+
+        echo $WORKSPACE
+        pwd
+
         # emptying the workspace
         mkdir -p empty
         rsync -r --delete empty/ ./
 
+        cat <<'EOF' > log-parser-plugin-rules.txt
```

Comment:
Lets please maintain this as separate file in the git repo.
I am fine with having something like `cp $automationrepo/$something/log-parser-plugin-rules.txt .` here though/